sambamba/pileup.d

```diff
@@ -610,6 +622,12 @@ void printUsage() {
     stderr.writeln("                    chunk size (in bytes)");
     stderr.writeln("         -B, --output-buffer-size=512_000_000");
     stderr.writeln("                    output buffer size (in bytes)");
+    stderr.writeln();
+    stderr.writeln("Sambamba paths:\n");
+    samtoolsInfo();
+    stderr.writeln("         samtools: ",samtoolsBin," ",samtoolsVersion);
```

Comment:
Please add spaces after punctuation to keep style consistent (in other places as well).